contracts/tgrade-validator-voting/src/contract.rs

```diff
@@ -56,8 +56,18 @@ pub fn execute(
         } => {
             // Migrate contract needs confirming that sender (proposing member) is an admin
             // of target contract
-            if let ValidatorProposal::MigrateContract { ref contract, .. } = proposal {
+            if let ValidatorProposal::MigrateContract {
+                ref contract,
+                ref migrate_msg,
+                ..
+            } = proposal
+            {
                 confirm_admin_in_contract(deps.as_ref(), &env, contract.clone())?;
+                dbg!(migrate_msg);
+                dbg!(migrate_msg.to_base64());
+                if migrate_msg.to_base64().is_empty() {
```

Comment:
Do you need the `.to_base64()`?
Just receiving it, it will have been parsed into raw bytes. We can check if those are empty, that's enough. (There should be a slightly simpler way to do the same check)

Reply:
That's correct, `to_base64()` here is actually superfluous